frontend/src/metabase/visualizations/components/FunnelNormal.jsx

```diff
@@ -200,12 +201,17 @@ const GraphSection = ({
   onVisualizationClick,
   className,
 }) => {
+  // Force SVG composite-layer until Safari bug is fixed: https://bugs.webkit.org/show_bug.cgi?id=257924
+  const compositeLayerStyles = isDesktopSafari()
```

Comment:
It would be a bit more consistent if you created `FunnelNormal.styled.tsx`, created the root element styles like 'const funnelRoot = props => css`...`' where props has `isNarrow`, `isSmall` transfer all the styles from the `div` on line 124, and added `will-change: transform` there

Reply:
Added your commit! Thanks